src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactoryTest.java

```diff
@@ -45,6 +45,7 @@ public void create() throws DocumentSelectorException {
             new ArrayList<>(),
             CcdBundlePaginationStyle.off,
             null,
```

Comment:
Do you want to add a test case where the document image isn't null?

Reply:
new test case created